app/components/modals/QRModal/QRModal.jsx

```diff
@@ -0,0 +1,27 @@
+import Modal from "../Modal";
+// XXX this isn't ideal, we should have modal specific css module!!!
+import styles from "../Modals.module.css";
```

Comment:
I'm planning to follow up with an immediate PR after this to split `Modals.module.css` to separate modals,
I'd avoid doing it as part of the QRs PR as the modals share some classes which should be refactored to be components - see comment about `.infoCloseButtonTop` in `Modals.module.css`.